app/models/manageiq/providers/redhat/inventory/persister/definitions/infra_group/networks_collections.rb

```diff
@@ -5,9 +5,12 @@ module ManageIQ::Providers::Redhat::Inventory::Persister::Definitions::InfraGrou
   def add_switches
     add_collection(infra, :switches) do |builder|
       if targeted?
-        arel = ::Switch.where(:uid_ems => manager_refs) if manager_refs.present?
-
-        builder.add_properties(:arel => arel) unless arel.nil?
+        # TODO are switches shared across emses? Seems like we weren't filling ems_id
```

Comment:
@agrare @borod108 so there is a bit disconnect between full and targeted. In full refresh we take switches related to hosts, in here we filter from all switches. Should we use :ems_id in Switch and make relation through that? Or should it always go through host?

Reply:
@Ladas I think for ovirt we should always go through the host for switches since ovirt doesn't have any shared switches yet